automation/check-patch.k8s-1.14.2-kind-sriov.sh

```diff
@@ -0,0 +1,242 @@
+#!/bin/bash -e
+set -x
+
+#############################################################
+# CI Notes TODO Move in a more appropriate location
+# Each CI node labeled as sriov-enable should have the following setup in place:
+# - have the vfio_pci module loaded with a modprobe.d rule
+# - since we use kind, in order to have the dns working we need to
+#   - have the br_netfilter module loaded with a modprobe.d rule
+#   - have the following lines on /etc/sysctl.conf (see https://stackoverflow.com/questions/48148838/kube-dns-error-reply-from-unexpected-source)
+#               net.bridge.bridge-nf-call-ip6tables = 1
+#               net.bridge.bridge-nf-call-iptables = 1
+#               net.bridge.bridge-nf-call-arptables = 1
+#
+#############################################################
+
+
+#############################################################
+# This is based on https://github.com/SchSeba/kubevirt-docker
+#############################################################
+
+export NO_PROXY="localhost,127.0.0.1,172.17.0.2"
+
+export WORKSPACE="${WORKSPACE:-$PWD}"
+readonly ARTIFACTS_PATH="${ARTIFACTS-$WORKSPACE/exported-artifacts}"
+
+CLUSTER_NAME=sriov-ci
+CLUSTER_CONTROL_PLANE=${CLUSTER_NAME}-control-plane
+CONTAINER_REGISTRY_HOST="localhost:5000"
+
+CLUSTER_CMD="docker exec -it -d ${CLUSTER_CONTROL_PLANE}"
+
+KUBEVIRT_PATH=`pwd`
+CLUSTER_DIR="cluster-up/cluster/k8s-1.14.2-kind-sriov"
+MANIFESTS_DIR="${CLUSTER_DIR}/manifests"
+
+
+function wait_containers_ready {
+    echo "Waiting for all containers to become ready ..."
+    kubectl wait --for=condition=Ready pod --all -n kube-system --timeout 12m
+}
+
+function wait_kubevirt_up {
+    echo "Waiting for kubevirt to be ready ..."
+    kubectl wait --for=condition=Ready pod --all -n kubevirt --timeout 12m
+    kubectl wait --for=condition=Ready pod --all -n cdi --timeout 12m
+}
+
+function wait_kind_up {
+    while [ -z "$(docker exec --privileged ${CLUSTER_CONTROL_PLANE} kubectl --kubeconfig=/etc/kubernetes/admin.conf get nodes --selector=node-role.kubernetes.io/master -o=jsonpath='{.items..status.conditions[-1:].status}' | grep True)" ]; do
+        echo "Waiting for kind to be ready ..."        
+	    sleep 10
```

Comment:
you can remove the sleep if you use the suggestion above

Reply:
nit: isn't there something wrong with indentation here?